controller/ControllerRTL.py

```diff
@@ -115,6 +118,13 @@ def construct(s, ControllerIdType, CmdType, FromCpuPktType, NocPktType,
     # on, the format can be packet-based or flit-based.
     s.recv_from_cpu_pkt //= s.recv_from_cpu_pkt_queue.recv
 
+    @update
+    def update_send_to_cpu_signal():
+        s.send_to_cpu_pkt.val @= 0

```

Comment:
Good job to initialize `s.send_to_cpu_pkt.val`, however, we also need to initialize `s.send_to_cpu_pkt.msg` to avoid latch during synthesis.